channels/uk.m3u

```diff
@@ -15,6 +19,10 @@ http://akaal.zecast.net/akaal-live/smil:akaaltv.smil/playlist.m3u8
 https://alaraby.cdn.octivid.com/alaraby/smil:alaraby.stream.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="AlhiwarTV.uk" tvg-name="Alhiwar TV" tvg-country="ARAB" tvg-language="Arabic" tvg-logo="https://i.imgur.com/ooiJwD5.png" group-title="",Alhiwar TV (1080p)
 https://mn-nl.mncdn.com/alhiwar_live/smil:alhiwar.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="Alibi.uk" tvg-name="Alibi" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/IlXokSN.png" group-title="Series",Alibi
+http://stream.tvtap.live:8081/live/alibi.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="AMC.uk" tvg-name="AMC" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/7wKiXrs.jpg" group-title="Entertainment",AMC
```

Comment:
Originates from the US.

Reply:
https://alibi.uktv.co.uk/ maybe let it here ?

Reply:
Okay... I also accept this one, but it's streamed in Europe : https://en.wikipedia.org/wiki/AMC_(European_TV_channel)

Reply:
Better to move the channel to the American playlist, just leaving `tvg-country="UK"`.